cmd/thanos/compact.go

```diff
@@ -458,6 +460,56 @@ func runCompact(
 		return cleanPartialMarked()
 	}
 
+	if conf.compactionProgressMetrics {
+		g.Add(func() error {
+			unRegisterer := &receive.UnRegisterer{Registerer: reg}
+			ps := compact.NewCompactionProgressCalculator(unRegisterer, tsdbPlanner)
+			var ds *compact.DownsampleProgressCalculator
+			if !conf.disableDownsampling {
+				ds = compact.NewDownsampleProgressCalculator(unRegisterer)
+			}
+
+			return runutil.Repeat(5*time.Minute, ctx.Done(), func() error {
+
+				if err := sy.SyncMetas(ctx); err != nil {
+					return errors.Wrapf(err, "could not sync metas")
+				}
+
+				originalMetas := sy.Metas()
+				groups, err := grouper.Groups(originalMetas)
+				if err != nil {
+					return errors.Wrapf(err, "could not group original metadata")
```

Comment:
We don't need to call it `orgininal`

Reply:
I mean in the log unnecessary to call it original. "Could not group metadata" is good enough

Reply:
This is still not solved @metonymic-smokey